app/src/js/components/home.js

```diff
@@ -204,6 +221,7 @@ class Home extends React.Component {
           </section>
 
           {this.renderButtonListSection(overview, 'Updates')}
+          {this.renderDistrubutionErrors(dist)}
           {this.renderButtonListSection(distErrorStats, 'Distribution Errors', 'distributionErrors')}
           {this.renderButtonListSection(distSuccessStats, 'Distribution Successes', 'distributionSuccesses')}
```

Comment:
Just thinking out loud here, but I wonder if it makes sense to somehow encapsulate rendering the error vs the distribution button sections into a single render function. Like, if there's errors, display that, and if not, display the two subsections?

Thinking that might make the html markup more logical. Maybe something to consider if we refactor this to a functional component.

Reply:
I thought about that, It's possible for both errors and stats to be shown (although I had to think hard at how that would be). And I opted for only displaying the distribution errors.  But it does seem like we could make a component out of the whole thing.

Reply:
Ah that makes it even more interesting. Well, something to consider later perhaps

Reply:
I do kind feel like we have some technical debt to get to.  I wonder if that will ever happen. I will peek at this again, but merge this now.  Thanks.